cmd/callrpc.go

```diff
@@ -0,0 +1,58 @@
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/ava-labs/avash/cfg"
+	pmgr "github.com/ava-labs/avash/processmgr"
+	"github.com/spf13/cobra"
+	"github.com/ybbus/jsonrpc"
+)
+
+// CallRPCCmd issues an RPC to a node endpoint using JSONRPC protocol
+var CallRPCCmd = &cobra.Command{
+	Use:     "callrpc [node name] [endpoint] [method] [JSON params] [var scope] [var name]",
+	Short:   "Issues an RPC to a node.",
+	Long:    `Issues an RPC to a node endpoint for the specified method and params.
+	Response is saved to the local varstore.`,
+	Example: `callrpc n1 ext/avm avm.getBalance {"address":"X-KqpU28P2ipUxfTfwaT847wWxyXB4XuWad","assetID":"AVA"} s v`,
+	Args: cobra.MinimumNArgs(6),
+	Run: func(cmd *cobra.Command, args []string) {
+		log := cfg.Config.Log
+		meta, err := pmgr.ProcManager.Metadata(args[0])
+		if err != nil {
+			log.Error("process not found: %s", args[0])
+			return
+		}
+		var md Metadata
+		if err = json.Unmarshal([]byte(meta), &md); err != nil {
+			log.Error("unable to unmarshal metadata for process %s: %s", args[0], err.Error())
+			return
+		}
+		jrpcloc := fmt.Sprintf("http://%s:%s/%s", md.Serverhost, md.HTTPport, args[1])
```

Comment:
We need to consider the case where the client uses https instead of http. 

See `--http-tls-enabled` flag on Gecko client.